pkg/api/v1/clients/file/resource_client_test.go

```diff
@@ -18,24 +19,40 @@ import (
 )
 
 var _ = Describe("Base", func() {
+
 	var (
-		client *ResourceClient
-		tmpDir string
+		client     *ResourceClient
+		tmpDir     string
+		namespace1 = "ns1"
+		namespace2 = "ns2"
 	)
+
 	JustBeforeEach(func() {
 		var err error
+
 		tmpDir, err = ioutil.TempDir("", "base_test")
 		Expect(err).NotTo(HaveOccurred())
+
+		// Create a directory per namespace inside the tmpDir.
```

Comment:
Because of the change we made below, to list resources in a specific namespace, instead of all namespaces, must must ensure that the namespace directories are created during the setup of the file resource client test.